source/extensions/transport_sockets/tls/stats.cc

```diff
@@ -14,6 +14,11 @@ SslStats generateSslStats(Stats::Scope& store) {
                         POOL_HISTOGRAM_PREFIX(store, prefix))};
 }
 
+CertStats generateCertStats(Stats::Scope& scope, std::string cert_name) {
+  std::string prefix_cert_name("ssl." + cert_name + ".");
+  return {ALL_CERT_STATS(POOL_GAUGE_PREFIX(scope, prefix_cert_name))};
```

Comment:
Please add tag-extraction rules for these certs here: https://github.com/envoyproxy/envoy/blob/cfdc99a00d43c6c08c22229c7053410446e51a3a/source/common/config/well_known_names.cc#L201

And add some tests for them

Reply:
Got it. Added